pkg/kclshim/example_input.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Mind moving all of the test data into `pkg/kclshim/fixtures`? It's a pattern used by other packages in eno